scripts/process_data.py

```diff
@@ -0,0 +1,234 @@
+
+# coding: utf-8
+
+# # Processing all datasets to be used in downstream analyses
+
+# In[1]:
+
+import os
+import requests
+import numpy as np
+import pandas as pd
+
+from sklearn import preprocessing
+
+
+# ## Define Input and Output Filenames
+
+# In[2]:
+
+# Input Files
+rna_file = os.path.join('data', 'raw', 'HiSeqV2')
+mut_file = os.path.join('data', 'raw', 'PANCAN_mutation')
+copy_file = os.path.join('data', 'raw', 'Gistic2_CopyNumber_Gistic2_all_thresholded.by_genes')
+clinical_file = os.path.join('data', 'raw', 'samples.tsv')
+
+
+# In[3]:
+
+# Output Files
+# Processing RNAseq data by z-score and zeroone norm
+rna_out_file = os.path.join('data', 'pancan_scaled_rnaseq.tsv')
+rna_out_zeroone_file = os.path.join('data', 'pancan_scaled_zeroone_rnaseq.tsv')
+
+# Mutation Data
+mut_out_file = os.path.join('data', 'pancan_mutation.tsv')
+
+# Two copy number matrices, for thresholded (2) gains and losses
+copy_gain_out_file = os.path.join('data', 'copy_number_gain.tsv')
+copy_loss_out_file = os.path.join('data', 'copy_number_loss.tsv')
+
+# Clinical data
+clinical_processed_out_file = os.path.join('data', 'clinical_data.tsv')
+
+# OncoKB output file
+oncokb_out_file = os.path.join('data', 'oncokb_genetypes.tsv')
+
+# Status matirix integrating mutation and copy number events
+known_status_file = os.path.join('data', 'status_matrix.tsv')
+
+
+# ## Load Data
+
+# In[4]:
+
+rnaseq_df = pd.read_table(rna_file, index_col=0)
+mutation_df = pd.read_table(mut_file)
+copy_df = pd.read_table(copy_file, index_col=0)
+clinical_df = pd.read_table(clinical_file, index_col=0, low_memory=False)
```

Comment:
Can remove the low memory option. See [here.](https://stackoverflow.com/questions/24251219/pandas-read-csv-low-memory-and-dtype-options)